armi/reactor/components/componentParameters.py

```diff
@@ -59,43 +59,20 @@ def getComponentParameterDefinitions():
             description="Component temperature in {}".format(units.DEGC),
         )
 
-        pb.defParam(
-            "temperatureInCBOL",
```

Comment:
I am actually quite happy to see this change.

But, I admit, from the PR description this change surprises me.

Reply:
lol we knew you would be pleased